daemon/src/cfd_metrics.rs

```diff
@@ -0,0 +1,116 @@
+use crate::projection::Cfd;
+use model::Position;
+use model::Usd;
+use rust_decimal::prelude::ToPrimitive;
+use std::collections::HashMap;
+
+const DIRECTION_LABEL: &str = "direction";
```

Comment:
🙃 `Direction` as term does not exist in the `itchysats` repo. I know we started using it in `metrics` and `automation`. I think we should agree on what terminology to use to avoid confusion in the future. In `itchysats` it is called `Position`. Is that not correct?

Reply:
Happy to go with `position` :)